Mathlib/Analysis/Calculus/Deriv/Mul.lean

```diff
@@ -169,7 +169,7 @@ lemma deriv_const_smul' {f : 𝕜 → F} {x : 𝕜} {R : Type*} [Field R] [Modul
     · simp only [zero_smul, deriv_const']
     · have H : ¬DifferentiableAt 𝕜 (fun y ↦ c • f y) x := by
         contrapose! hf
-        change DifferentiableAt 𝕜 (fun y ↦ f y) x
+        change DifferentiableAt 𝕜 f x
```

Comment:
Does this still do anything now? If not, remove `change` entirely.

Reply:
Good point, removed!